src/screens/ProfileScreen/index.js

```diff
@@ -739,7 +761,7 @@ const styles = StyleSheet.create({
     flexDirection: 'column',
     paddingHorizontal: 20,
     backgroundColor: COLORS.white,
-    marginTop: 14
+    paddingVertical: 11
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The padding of the container has been modified. Make sure this doesn't break the existing UI layout.
```diff
-    marginTop: 14
+    paddingVertical: 11
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->